Polli/Reinicio.py

```diff
@@ -0,0 +1,115 @@
+from dataset_alunas import dataset
+
+dataset
+
+def main():
+    print("\n---  Seja bem vinda a Escola do Reprograma!  ---")
+    print("Sistema de informações de alunas")
+
+    while True:
+        cod_opcao = obter_opcao()
+        
+        if cod_opcao == 1:
+            incluir_nova_aluna()
+        elif cod_opcao == 2:
+            consultar_lista_alunas()
+        elif cod_opcao == 3:
+            consultar_faltas_aluna()
+        elif cod_opcao == 4:
+            consultar_notas_aluna()
+        elif cod_opcao == 5:
+            consultar_status_aprovacao()
+        elif cod_opcao == 6:
+            print("Encerrando o programa...")
+            break
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+    return codigo_opcao
+
+    
+def incluir_nova_aluna():
+    print("Insira os seguintes dados: ")
+    nome = str(input("Nome da aluna: ")) #Recebo nome da aluna
+    sobrenome = str(input("Sobrenome da aluna: ")) #Recebo sobrenome da aluna
+    turma = int(input("Turma da aluna (nº): ")) #Recebo nome a turma
```

Comment:
Dentro do dataset, nosso tipo de dado que está dentro de Turma é string. Conseguimos olhar isso porque no dicionário temos:
"Turma": "Turma A"
"Turma A" é uma string, pois é um texto. Se ela fosse um int, deveria ser apenas numérico :)

**Sugestão:**
```suggestion
    turma = input("Turma da aluna (nº): ") #Recebo nome a turma
```